agent/util-scripts/pbench-tool-meister-start

```diff
@@ -136,6 +136,14 @@ from pbench.agent.utils import (
 from pbench.common.utils import validate_hostname
 
 
+# The --orchestrate parameter default choice, and the full list of choices.
+_orchestrate_choices = ["create", "existing"]
+_def_orchestrate_choice = "create"
+assert _def_orchestrate_choice in _orchestrate_choices, (
+    f"Logic Bomb!  The default orchestrate choice, {_def_orchestrate_choice!r},"
+    f" is not one of the available choices, {_orchestrate_choices!r}"
+)
+
```

Comment:
At this point I'm not even going to make an off-hand comment that this is starting to look like an ENUM without the semantic rigor. I didn't say that out loud, did I? ;-)

Reply:
Tracked in #2316.